Source of the arcticle exposed as a hyperlink in title

fourier opened the pull request:
1) Fixed logic error when after the first invalid RSS feed didn't process others
2) Source of the article is now a hyperlink in header

fourier committed:
1) Fixed logic error when after the first invalid RSS feed didn't process others; 2) Added Source to the begining of the formatted post to allow to open long posts in a separate window/tab instead of opening in diaspora post and scrolling down to the end looking for the source

fourier committed:
Source is now in title like in HackerNews bot

fourier committed:
Source is now in title like in HackerNews bot

fourier committed:
merged with upstream

fourier committed:
fixed conflict

Debakel commented on feedDiasp/Diasp.py:
```diff
@@ -2,35 +2,42 @@
-		if append is not None:
-			text += '  \n' + append
-		self.stream.post(text, provider_display_name=self.provider_name)
+  def __init__(self, pod=None, username=None, password=None, provider_name=''):
```
Please keep the tab as indentation method, so i can easier identify you additions/deletions.
Thank you!

fourier committed:
Updated diaspy to the latest version

Commit:
fix post creation

at this point post is initialized for the first time. Otherwise you get a "local variable 'post' referenced before assignment" if you call post without a title

fourier committed:
Merge pull request #1 from schiesbn/patch-1

fix post creation
